include/ddc/detail/tagged_vector.hpp

```diff
@@ -401,6 +401,12 @@ class TaggedVector : public ConversionOperators<TaggedVector<ElementType, Tags..
         ((m_values[type_seq_rank_v<Tags, tags_seq>] *= rhs.template get<Tags>()), ...);
         return *this;
     }
+
+    template <class OElementType>
+    explicit operator TaggedVector<OElementType, Tags...>()
```

Comment:
Looking at the constructors, there should already be one conversion constructor. It is not selected because it takes input parameters by `const&` thus disabling the conversion from a `Chunk<T> const&` to a `ChunkSpan<T>` (only `ChunkSpan<const T>` is allowed from a `Chunk<T> const&`).

I still need to think if it is compatible with the rest of the API. Is it urgent ?

Reply:
No I can use a workaround in the meantime

Reply:
since we can do it, offering a conversion constructor should be preferred to a conversion operator.

What about offering `template <class OElementType> TaggedVector::TaggedVector(TaggedVector<OElementType, Tags...>)`?

Reply:
The conversion constructor already exists:
https://github.com/Maison-de-la-Simulation/ddc/blob/d401a6d36477775dbca108a6490c5a119cb99ee5/include/ddc/detail/tagged_vector.hpp#L260-L265

Reply:
but it takes a `cont &` and this is not a solution in your case. We could add a non-const ref one

Reply:
Would it not solve the issue?

Reply:
Fixed. I think this fixes the issue. But I don't have the problem anymore as I changed strategy and ended up writing the `VectorField` class

Reply:
Thanks, looks good to me, @tpadioleau ?